src/test/test_verify_multithreading.cpp

```
@@ -3,6 +3,17 @@
 
 #include "test_verify.hpp"
 
+static prevail::Program prepare(const std::string& section) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)